docs/2.0/manual.md

```diff
@@ -437,22 +440,26 @@ Pheniqs constructs new segments for the output read and must generate correspond
 {: .example}
 
 # Pass filter reads
-Some reads are marked by the sequencing platform as not passing the vendor quality control. For instance Illumina sequencers perform an internal [quality filtering procedure](http://support.illumina.com/content/dam/illumina-marketing/documents/products/technotes/hiseq-x-percent-pf-technical-note-770-2014-043.pdf) called chastity filter, and reads that pass this filter are called **PF** for **pass-filter**. This can be [signaled](https://en.wikipedia.org/wiki/FASTQ_format#Illumina_sequence_identifiers) on the comment portion of the read identifier in [FASTQ](glossary.html#fastq) files or the **512** flag on a SAM record flag. Setting this attribute to **true** will instruct Pheniqs to decode those unfiltered reads and include them in the output. If the attribute is set to **false** those reads are dropped. `include filtered` defaults to **false**.
+Some reads are marked by the sequencing platform as not passing the vendor quality control. For instance, Illumina sequencers perform an internal [quality filtering procedure](http://support.illumina.com/content/dam/illumina-marketing/documents/products/technotes/hiseq-x-percent-pf-technical-note-770-2014-043.pdf) called chastity filter, and reads that pass this filter are called **PF** for **pass-filter**. This can be [signaled](https://en.wikipedia.org/wiki/FASTQ_format#Illumina_sequence_identifiers) on the comment portion of the read identifier in [FASTQ](glossary.html#fastq) files or the **512** flag on a SAM record flag. Setting this attribute to **true** will instruct Pheniqs to decode those unfiltered reads and include them in the output. If the attribute is set to **false** those reads are dropped. `include filtered` defaults to **false**.
+
+**I think this would be clearer if the key was "include only pf reads" or something like that. When I talk about filtered reads I generally refer to those that passed a particular filter, not those that did not pass.**
```

Comment:
This mimics the behavior of all other tools I have seen. By default PF reads are not included. Setting this variable to true will include them but the flag itself defaults to false. Since any flag in pheniqs that is entirely missing from the config is assumed to be false that means that completely ignoring this option yields the expected behavior. we can potentially change the name of the variable but forcing the user to explicitly set a variable to get the most common and expected behavior seems counter intuitive to me...

Reply:
That's fair, just seemed a little confusing while reading it